RELEASING.md

```diff
@@ -1,5 +1,42 @@
 # Releasing
 
+- [ ] Make sure all important / big PRs have been merged by now
+- [ ] Create a news item draft PR on [akka.github.com](https://github.com/akka/akka.github.com), using the milestone
+- [ ] Make sure to update `versions.json` in it
+- [ ] Make sure any running [actions](https://github.com/akka/akka/actions) for the commit you would like to release have completed.
+- [ ] Tag the release `git tag -a -s -m 'Release v$VERSION$' v$VERSION$` and push the tag `git push --tags`
+- [ ] Check that the GitHub Actions release build has executed successfully (it should publish artifacts to Sonatype and documentation to Gustav)
+- [ ] Check [reference](https://doc.akka.io/docs/akka/$VERSION$/) documentation
+- [ ] Check the release on [Maven central](https://repo1.maven.org/maven2/com/typesafe/akka/akka-actor_2.13/$VERSION$/)
```

Comment:
shall we turn this into a release issue template as we have in some other places?

Reply:
Yes, this was intended to be a first step in that direction ;) - I can split it out already